homeassistant/components/zwave_js/services.py

```diff
@@ -413,6 +409,14 @@ async def async_multicast_set_value(self, service: ServiceCall) -> None:
         nodes = service.data[const.ATTR_NODES]
         broadcast: bool = service.data[const.ATTR_BROADCAST]
 
+        if not broadcast and len(nodes) == 1:
+            const.LOGGER.warning(
+                "Passing the service call to the zwave_js.set_value service since "
+                "only one node was targeted"
+            )
+            await self.async_set_value(service)
```

Comment:
Does the service data validation of the current service match the set value service?

Reply:
The multicast service has more validation in place for multicast specific things. The `set_value` service also has a `wait_for_result` parameter that can force the library to either wait or not wait for the response of the command. If we want the schemas to match, we can probably add a `wait_for_result` parameter to the multicast service call, but I think it's a no-op for the multicast command (although I thought I read that in the docs and I can't find any reference to that)

Reply:
If we don't use the service registry to call the service handler we need to make sure that the service data is still valid when it reaches the service handler. If we know that it is valid it's ok.

Reply:
Ah, yes it is valid. I went for this approach because we popped the device and entity IDs so the schema validation would fail if we went through a normal service call (requires at least one, doesn't expect the nodes).